CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # CHANGELOG
 
 -----
+## `v3.2.0-beta`
```

Comment:
storage has plenty of changes. Please add them too. You can email me offline if you like, we can work on a changelog for the latest changes in there. but please don't omit them in the changelog.